### ISSUE TITLE
Add more details to GPS trace overview

### ISSUE THREAD

**Issue by hikemaniac** [openstreetmap/openstreetmap-website#1980]
I would find it useful to get more details on a GPS traces page such as:
- Start time
- Length (In miles or kilometers)
- End coordinate
- Maximum and minimum altitude (where present)
- Geocoded location

and maybe the traces file size.

This data might be really useful for picking the right trace for mapping and also detecting vandalism. What do you think about it? Are these things even possible to add and would that be in scope?

**Comment by tomhughes** [comment]
Why might it be useful? Isn't the normal way of mapping against GPS to use the point cloud rather than specific traces? Well except maybe mapping against one of your own traces.

**Comment by jguthrie100** [comment]
This is one thing I've always questioned when it's been brought up in the past and the 'we don't need extra data' line is brought out.

Surely the more data, the better? Basic things such as length, location, max altitude are just generally useful information and doesn't detract from the validity or appropriateness of the page in any way whatsoever!

And if those bits of data are so useless, then why even bother with the trace route image? It's not as if you can use the thumbnail to accurately determine anything anyway!

**Comment by tomhughes** [comment]
Well it's one thing for somebody to offer a patch to add them, it's another to ask people to divert time that might be better used to such a thing.

Given our exceedingly limited resources we need to try and target them where they can do most good.

It's also a fallacy that features are "free" even if the implementation cost is ignored - they add maintenance overhead and become something that is hard to remove in the future if for some reason they are no longer practical to maintain.

**Comment by tomhughes** [comment]
In any case it was the reporter that said it might prove useful, so I assumed he had a use case in mind and I was reallyt just asking them to explain that use case in more detail so that we can better understand what is being requested.

**Comment by hikemaniac** [comment]
> Why might it be useful? Isn't the normal way of mapping against GPS to use the point cloud rather than specific traces?

That's true and is also what I use most of the time, but the GPS track overlay has proven unreliable, some traces appear as dotted lines others are not shown at all, even though they are public and since a few days new traces [aren't rendered at all](https://help.openstreetmap.org/questions/64178/public-gps-traces-map-layer-not-displaying-latest-traces). Then it is desirable to pick traces from your profile, to provide the needed data and these additional details would make picking the right one much easier.

> Basic things such as length, location, max altitude are just generally useful information and don't detract from the validity or appropriateness of the page in any way whatsoever!

A long trace with lots of squiggles and twitches appearing on the preview picture can be [GPS track vandalism](https://wiki.openstreetmap.org/wiki/File:Gpx_graffiti_vandalism.png). If it is just a short sequence it might also just be normal GPS inaccuracy. The other bullet points are indeed only needed to quickly choose a track from your own profile, matching the location.

> It's one thing for somebody to offer a patch to add them, it's another to ask people to divert time that might be better used to such a thing.

I am not trying to push anything, I just opened the ticket to hear if other people might also find this a helpful addition and if it is in scope for the project. I believe that this issue tracker is more like a to-do list and if someone has spare time and thinks this is useful he might provide code for this, even though the ticket can as well be ignored for years.
I have no problems with closing the issue if you find it totally useless, I'm just collecting ideas.

**Comment by tomhughes** [comment]
I'm talking about the point cloud downloaded from the API not the rendered tiles, but if the rendered tiles aren't being updated then somebody needs to tell us so we can look at it!

**Comment by tomhughes** [comment]
Trace feed was fixed in https://github.com/openstreetmap/openstreetmap-website/commit/ba05009de1e64e8d842608f8dcc6a5ee8c6e740f and the rendered tiles should be catching up now.

**Comment by hikemaniac** [comment]
Thank you for fixing this! Everything appears fine now.

Anyways, the point cloud would most definitely be the perfect solution, however I mostly use iD for editing and I can only access the rendered tiles and specific tracks by pressing "edit" on the GPS trace overview.
This means that these additional details can be useful to users of the iD editor, but it is a low-priority issue. 😉
